asciimatics/widgets/textbox.py

```diff
@@ -283,6 +301,35 @@ def _reflowed_text(self):
 
         return self._reflowed_text_cache
 
+    @property
+    def hide_cursor(self):
+        """
+        Set to True to stop the cursor from showing. If the cursor is hidden,
+        scrolling will move the entire TextBox instead of just the cursor.
```

Comment:
This will also need updating if you move the scroll logic to your custom widget.